.github/workflows/find.yml

```diff
@@ -120,6 +120,7 @@ jobs:
       OPENSEARCH_HOST: localhost
       OPENSEARCH_PASSWORD: find.PASS123
       OPENSEARCH_USE_SSL: false
+      EMBEDDING_API_KEY: ThisIsAnExampleKeyForTestPurposeOnly
```

Comment:
I think this setting should only be set in the tests where it's used, not globally.